docs/docsite/rst/porting_guides/porting_guide_2.7.rst

```diff
@@ -17,7 +17,14 @@ This document is part of a collection on porting. The complete list of porting g
 Playbook
 ========
 
-No notable changes.
+Role Precedence Fix during Role Loading
+---------------------------------------
+
+Ansible 2.7 makes a small change to variable precedence when loading roles, to match the :ref:`variable precedence expectations <ansible_variable_precedence>`.
```

Comment:
Aren't we fixing a bug? Maybe indicate we made a change in order to bring variable precedence in line with documentation.